### PR TITLE
Move membership banner for US audience out of A/B tests

### PR DESCRIPTION
**Pull request by regiskuckaertz**
As requested in https://github.com/guardian/frontend/pull/11158, the banner for US readers has been removed from A/B tests and merged with the similar banner for readers of the UK edition.

### CODE REVIEW

File: static/src/stylesheets/module/_release-message.scss [hunk 1]
```diff
@@ -432,7 +432,8 @@ $btn-height: gs-height(1) - $gs-baseline/2; //30px
 .site-message--membership-message,
 .site-message--membership-message-uk,
 .site-message--membership-message-uk-redisplayed,
-.site-message--membership-message-usa {
+.site-message--membership-message-us,
+.site-message--membership-message-us-redisplayed {
```
Comment (uplne) [inline static/src/stylesheets/module/_release-message.scss]: Do we need to have these class names for each edition? I believe the templates have identical styling so they should be good with just generic class names `.site-message--membership-message` and `.site-message--membership-message-redisplayed`.

Comment (uplne) [inline static/src/stylesheets/module/_release-message.scss]: You might also want to remove switch for this AB test. AB tests are always behind the switch.

Comment (regiskuckaertz) [inline static/src/stylesheets/module/_release-message.scss]: Will do. Actually even the `-redisplayed` class is redundant.

**Comment by uplne** [comment]
:+1:

**Pull request merged**